languages/java/exercises/concept/chars/src/test/java/IdentifierTest.java

```diff
@@ -0,0 +1,60 @@
+import org.junit.Test;
+import org.junit.Ignore;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class IdentifierTest {
+
+    @Test
+    public void empty() {
+        assertThat(Identifier.clean("")).isEmpty();
+    }
+
+    @Ignore("")
```

Comment:
This advice is for all `@Ignore` annotations in the file.
To be aligned to Java track style, we use this `@Ignore("Remove to run test")`